gr-analog/python/analog/qa_agc.py

```diff
@@ -473,6 +473,8 @@ def test_006(self):
         dst_data = dst1.data()
         M = 100
         result = [abs(x) for x in dst_data[N - M:]]
+        for idx, (x, y) in enumerate(zip(result, M * [ref, ])):
+            self.assertAlmostEqual(x, y, 4, f"failed at pos {idx}")
```

Comment:
This looks like it's duplicating the check on the following line (with a friendlier error message). If that's the case, we could delete the next assertion.

Also, this could be simplified:
```suggestion
        for idx, x in enumerate(result)):
            self.assertAlmostEqual(x, ref, 4, f"failed at pos {idx}")
```

Reply:
did the simplification.